Hackney.Shared.HousingSearch.Tests/Domain/Tenure/TenureTests.cs

```diff
@@ -0,0 +1,19 @@
+using FluentAssertions;
+using Hackney.Shared.HousingSearch.Tests.Helper;
+using Xunit;
+
+namespace Hackney.Shared.HousingSearch.Tests
+{
+    public class TenureTests
+    {
+
+        [Fact]
+        public void TenureHasPropertiesSet()
+        {
+            Domain.Tenure.Tenure thisTenure = Constants.ConstructTenureFromConstants();
```

Comment:
probably worth adding using for `Domain.Tenure.Tenure` to make it easier to type

Reply:
We've tried to do this a few ways yesterday, but the _using_ directive seems to mix things up here so haven't found a way to do this (moving the path to _using_) yet. Open to suggestions.

Reply:
Actually going to use what @Duslerke suggested [here](https://github.com/LBHackney-IT/housing-search-shared/pull/73#discussion_r1571009240)